pmg/models/soundcloud_track.py

```diff
@@ -11,10 +11,7 @@
 logger = logging.getLogger(__name__)
 
 SOUNDCLOUD_ARTWORK_PATH = "pmg/static/resources/images/logo-artwork.png"
-SOUNDCLOUD_AUTH_500_MSG = (
-    "500 Server Error: Internal Server Error for url:"
-    " https://api.soundcloud.com/oauth2/token"
-)
+SOUNDCLOUD_AUTH_500_MSG = "500 Server Error: Internal Server Error for url"
```

Comment:
This is relaxing the constraint - I think it's fine to ignore 500s for that URL, but not for all. So let's keep the URL in there.